django_reference_app/tests/test_homepage.py

```diff
@@ -0,0 +1,6 @@
+from django import test
+
+class URLTests(test.TestCase):
+  def test_homepage(self):
+    response = self.client.get('/')
```

Comment:
PEP8 dictates that [there should be 4 spaces per indentation level](https://www.python.org/dev/peps/pep-0008/#indentation).

Reply:
@kerin as per our discussion, I'm going to add a ticket to add auto-linting as a required phase of the overall test suite.  You OK if we address these as part of that ticket?

Reply:
yep 👍